.github/workflows/social-media-automation.yml

```
@@ -0,0 +1,98 @@
+        - all
+        - linkedin
+        - twitter
+        - instagram
+        - reddit
+
+jobs:
+  social-media-posting:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)